pallets/delegation/src/benchmarking.rs

```diff
@@ -26,7 +26,7 @@ use frame_support::{
 use frame_system::RawOrigin;
 use sp_core::{offchain::KeyTypeId, sr25519};
 use sp_io::crypto::sr25519_generate;
-use sp_runtime::MultiSignature;
+use sp_runtime::{traits::Zero, MultiSignature};
```

Comment:
why was this added?

Reply:
This was a warning I caught, but it has been also addressed in https://github.com/KILTprotocol/mashnet-node/pull/265.